src/test/java/seedu/studybananas/model/task/TaskTest.java

```diff
@@ -80,4 +80,9 @@ public void isDateTimeOverlapped() {
         // Task without overlapped dateTime -> returns false
         assertFalse(CS2103T_WEEK8_QUIZ.isDateTimeOverlapped(CS2100_TUTORIAL_HOMEWORK));
     }
+
+    @Test
+    public void isLongerThanAnHour() {
```

Comment:
Not sure if you are continuing with this test?

Reply:
Yup I forgot to switch to another branch for this